server/service/src/sync/translations/item.rs

```diff
@@ -69,7 +69,9 @@ fn to_legacy_ven_category(ven_category: VENCategory) -> String {
 }
 
 pub(crate) fn ordered_simple_json(text: &str) -> Result<String, serde_json::Error> {
-    let json: serde_json::Value = serde_json::from_str(text)?;
+    let mut json: serde_json::Value = serde_json::from_str(text)?;
+    // Saving dose_picture 'picture' type as incoming 'empty string' causes issues in integration tests
+    json.as_object_mut().unwrap().remove("dose_picture");
```

Comment:
```suggestion
       if let Some(json) = json.as_object_mut() {
        json.remove("dose_picture");
    }
```
May not matter but this feels safer, if we're very confident then would like a comment as to why it's safe.

https://github.com/msupply-foundation/open-msupply/wiki/Code-Review-Comments---Rust#dont-use-unwrap-unless-in-tests

Reply:
Thought to check in vs code and committed there: dea05985d5544bbe99a79242a10a50349a936dc1

Thanks so much for the catch